### ISSUE TITLE
option mode to export to amd like requirejs format

### ISSUE THREAD

**Issue by alexsad** [davidjamesstone/superviews.js#20]
Support to amd format
**format=amd**
```
define(["exports",'incremental-dom'], function (exports,IncrementalDOM) {
    "use strict";
    var myWidget = (function () {
        function myWidget(data) {
            IncrementalDOM.elementOpen('div');
            IncrementalDOM.text('welcome, '+data.name);
            IncrementalDOM.elementClose('div');
        }
        return myWidget;
    }());
    exports.myWidget = myWidget;
});

```

**Comment by alexsad** [comment]
Thanks, to accept the merge request!!!
A question, when we can get the version in npm with that new feature ?

**Comment by davidjamesstone** [comment]
No problem - thanks for the PR!

Just published to npm@2.0.1.

Cheers

Dave

**Comment by alexsad** [comment]
thanks, again great job!!!
